src/modalities/conversion/gpt2/conversion_model.py

```diff
@@ -42,6 +42,8 @@ def convert_model_config(modalities_config: dict) -> GPT2Config:
     config = modalities_config["model_raw" if "model_raw" in modalities_config else "model"]["config"]
     _check_conversion_criteria(config)
 
+    ffn_norm_key = "ffn_norm" if "ffn_norm" in config else "ffn_norm_config"
```

Comment:
a comment that this is due to a gpt2llm refactoring and ensure backwards compatibility.